src/main/java/org/junit/rules/Timeout.java

```diff
@@ -143,16 +143,17 @@ protected final boolean getLookingForStuckThread() {
      * @since 4.12
      */
     protected Statement createFailOnTimeoutStatement(
-            Statement statement) throws Exception {
+            Statement statement, Description description) throws Exception {
```

Comment:
We can't change the behavior or signature of the existing method. We could deprecate it (but not change or break it) and remove it later in accordance with our depreciation policy.

Often we get around this problem by creating a new protected or package scope method, and having the new method call the previous one. Sunglasses can override the new method. Note the Builder could create insrltances of a subclass of `Timeout`